aepp/exportDatasetToCloudStorage.py

```diff
@@ -0,0 +1,345 @@
+            self.logger.info("sourceConnectionId " + source_connection_id + " has been created.")
+            return source_connection_id
+        except KeyError:
+            raise RuntimeError(f"Error when creating source connection: {source_res}")
+
+    def createTargetConnection(self, base_connection_id: str, compression_type: str, data_format: str, export_path: str) -> str:
+        """
+        Create a target connection for Data Landing Zone Destination
+        Arguments:
+            base_connection_id : REQUIRED : The base connection id created in previous step
+            compression_type: REQUIRED : define compression type to use when saving data to file. Possible values:
+                gzip
+                none
+            data_format : REQUIRED : define the data format . Possible values:
+                json
+                parquet
+            export_path : REQUIRED : define the folder path in your dlz container
+        Returns:
+          target_connection_id(str)
+        """
+        target_res = self.flow_conn.createTargetConnection(
+            data={
+                "name": f"[CMLE][Week2] Data Landing Zone target connection created by {self.username}",
+                "baseConnectionId": base_connection_id,
+                "params": {
+                    "mode": "Server-to-server",
+                    "compression": compression_type,
+                    "datasetFileType": data_format,
+                    "path": export_path
+                },
+                "connectionSpec": {
+                    "id": self.DLZ_CONNECTION_SPEC_ID,
+                    "version": "1.0"
+                }
+            }
+        )
+        try:
+            target_connection_id = target_res["id"]
+            self.logger.info("targetconnectionId " + target_connection_id + " has been created")
+            return target_connection_id
+        except KeyError:
+            raise RuntimeError(f"Error when creating target connection: {target_res}")
+
+    def createFlow(self, source_connection_id: str, target_connection_id: str, on_schedule: bool) -> str:
+        """
+        Create a data flow for Data Landing Zone Destination
+        Arguments:
+          source_connection_id : REQUIRED : The source connection id created in previous step
+          target_connection_id : REQUIRED : The target connection id created in previous step
+          on_schedule: REQUIRED: define whether you would like to have the export following a fixed schedule or not
+        Returns:
+          dataflow_id(str)
+        """
+        if on_schedule:
+            schedule_params = {
+                "interval": 3,
+                "timeUnit": "hour",
+                "startTime": int(time.time())
+            }
+        else:
+            schedule_params = {
+                "interval": 1,
+                "timeUnit": "day",
+                "startTime": int(time.time() + 60*60*24*365)
+            }
+
```

Comment:
so here maybe instead of dataclasses cant we create a custom object that is json backed , we're still building json in code which is a bit messy, what I would like to see is to instantiate an object that has annotations containing mutator and accessor methods for flowspec and all the other fields